src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -4099,7 +4185,8 @@ address MacroAssembler::zero_words(Register ptr, Register cnt) {
 
 // base:  Address of a buffer to be zeroed, 8 bytes aligned.
 // cnt:   Immediate count in HeapWords.
-void MacroAssembler::zero_words(Register base, u_int64_t cnt) {
```

Comment:
looks like this edit was accidentially added to the backport, other than that - looks good

Reply:
Thx! That's right, this line break was inserted by mistake. Fixed.